### PR TITLE
nativefier 7.0.1 (new formula)

### PR DESCRIPTION
**Pull request by weihanglo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

[Nativefier](https://github.com/jiahaog/nativefier) is a command line tool that allows you to easily create a desktop application for any web site with succinct and minimal configuration.

### CODE REVIEW

File: Formula/nativefier.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"nativefier", "--version"
```
Comment (woodruffw) [inline Formula/nativefier.rb]: Any potential for a more substantive test?

Comment (weihanglo) [inline Formula/nativefier.rb]: Got it. I will add some tests.

Comment (weihanglo) [inline Formula/nativefier.rb]: Unfortunately, due to its dependency  [node-progress](https://github.com/visionmedia/node-progress), **nativefier** cannot run in non-tty mode such as `brew test`. I cannot add any meaningful test out of tty mode. There are some [pull requests](https://github.com/visionmedia/node-progress/pull/139) waiting for reply to fix this issue. Still, **nativefier** works well in tty mode.

Comment (woodruffw) [inline Formula/nativefier.rb]: That's fine - thanks for checking!

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by woodruffw** [comment]
Thanks for adding a new formula @weihanglo! :tada:

**Pull request closed without merging**